server/src/server.ts

```diff
@@ -0,0 +1,55 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+import fs from "fs";
+import { parse, ParsedQuery } from "query-string";
+
+const port = 4000;

```

Comment:
Create separate config file for infrastructure variables

Reply:
Moved to a separate file